jobs/pacman-data-shipper/src/main/java/com/tmobile/cso/pacman/datashipper/dao/RDSDBManager.java

```diff
@@ -207,4 +213,43 @@ public static void insertRecord(String tableName, Map<String, String> data) {
             LOGGER.error("Error Executing Query", e);
         }
     }
+
+    public static boolean isFirstTimePluginExecuted(String dataSource, List<PolicyTable> policyList) {
+        String recordsCountQuery = "SELECT COUNT(*) AS count FROM cf_PolicyTable WHERE assetGroup = ?";
+        try (Connection conn = getConnection();
+             PreparedStatement preparedStatement = conn.prepareStatement(recordsCountQuery)) {
+            preparedStatement.setString(1, dataSource);
+            return executeCountQuery(preparedStatement) == 0;
```

Comment:
I think the return value and method name are contradicting each other. We need to do
  return executeCountQuery(preparedStatement) > 0;
to match the method name, because count will >0 after executing plugin first time

Reply:
Changed method name to simplified and understandable one and changed condition to make relevant to method name .please check latest push